docs/kratos/guides/deploy-kratos-example.mdx

```diff
@@ -0,0 +1,631 @@
+Type=simple
+Restart=always
+RestartSec=1
+User=kratos
+ExecStart=/opt/kratos/bin/kratos -c /opt/kratos/config/kratos.yml serve
+
+[Install]
+WantedBy=multi-user.target
+```
+
+To run Ory Kratos using systemd add the systemd service to startup:
+
+```bash
+systemctl enable kratos.service
+Created symlink /etc/systemd/system/multi-user.target.wants/kratos.service → /etc/systemd/system/kratos.service.
+```
+
+Start kratos service using systemd:
+
+```
+systemctl start kratos.service
+```
+
+Check running processes with `ps ax | grep kratos`. If everything worked
+correctly you should see output like this:
+
+```bash
+19191 ?        Ssl    0:00 /opt/kratos/bin/kratos -c /opt/kratos/config/kratos.yml serve
+19206 ?        Ss     0:00 postgres: 12/main: kratos kratos 127.0.0.1(36094) idle
+```
+
+Check if it's accessible (if your DNS record is not set yet (it takes up to 24h
+to propagate), you can also send the request here directly to your droplet IP)
+
+```bash
+curl -s  http://accounts.example.com:4433/sessions/whoami | jq
+{
+  "error": {
+    "code": 401,
+    "status": "Unauthorized",
+    "reason": "No valid session cookie found.",
+    "message": "The request could not be authorized"
+  }
+}
+```
+
+We have Ory Kratos up and running, but we need to configure a reverse proxy to
+make the Kratos Admin API inaccessible via the public internet. We need to set
+`serve.public.host` and `serve.admin.host` to `127.0.0.1` to ensure Ory Kratos
+is listening on the loopback interface.
+
+Change the following sections in your `kratos.yml` (in case you forgot: you can
+find it in `/opt/kratos/config`):
+
+```diff
+serve:
+  public:
+    base_url: http://127.0.0.1:4433/
++    host: 127.0.0.1
+    cors:
+      enabled: true
+  admin:
+    base_url: http://kratos:4434/
++    host: 127.0.0.1
+```
+
+and then restart Kratos by running
+
+```bash
+service kratos restart
+curl -I http://accounts.example.com:4433
+curl: (7) Failed to connect to accounts.example.com port 4433: Connection refused
+```
+
+## Installing and configuring Nginx
+
+We'll use Nginx as reverse proxy and load balancer for our service. You can use
+another reverse proxy and load balancer instead. Install Nginx by running:
+
+```bash
+apt-get nginx certbot python3-certbot-nginx
+```
+
+Create a default configuration for the virtual host. To do this create the file
+`accounts.example.com` at `/etc/nginx/sites-available/` with the following
+content
+
+```bash
+cd /etc/nginx/sites-available/
+nano accounts.example.com
+```
+
+```
+server {
+        listen 80;
+        server_name accounts.example.com;
+}
+```
+
+We need to create a symlink to the `sites-enabled` directory
+
+```bash
+ln -s /etc/nginx/sites-available/accounts.example.com /etc/nginx/sites-enabled/accounts.example.com
+```
+
+Configure SSL via Certbot. Run the following command and answer the questions to
+set it up. When prompted choose to redirect HTTP traffic to HTTPS.
+
+```bash
+certbot --nginx -d accounts.example.com
```

Comment:
Got until here without any problems. Some error in my Nginx configuration now, but it is probably an error on my side. 
Will see to complete it tomorrow, the Nginx part is a bit more tricky/errorprone than installing and setting up Kratos 😁